src/Pages/List/Components/RoomCardList.js

```diff
@@ -1,20 +1,25 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import RoomCard from './RoomCard';
 import styled from 'styled-components';
 
-const RoomCardList = () => {
-  const [roomList, setRoomList] = useState();
-  useEffect(() => {
-    fetch('/data/roomcard.json')
-      .then(res => res.json())
-      .then(data => setRoomList(data.room_info));
-  }, [setRoomList]);
-
+const RoomCardList = ({
+  roomList,
+  selectedRoom,
+  setSelectedRoom,
+  calculateAvgRating,
+}) => {
   return (
     <StyledList>
       {roomList &&
         roomList.map(room => {

```

Comment:
```suggestion
      {roomList?.map(room => {
```
이런 것 [optional chaining](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Optional_chaining) 사용하시면 간결하게 표현 가능합니다!
어떻게 동작하는지는 문서 한번 보고 이해해보세요!